<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoteStepDialog</class>
 <widget class="QDialog" name="DemoteStepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>105</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>155</width>
    <height>105</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>155</width>
    <height>105</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Specify position</string>
  </property>
  <widget class="QPushButton" name="okButton_">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>76</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton_">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>77</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="buttonGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>107</width>
     <height>65</height>
    </rect>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton2">
      <property name="text">
       <string>split parent</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton3">
      <property name="text">
       <string>after parent</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton1">
      <property name="text">
       <string>before parent</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup1</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>okButton_</tabstop>
  <tabstop>cancelButton_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>DemoteStepDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton_</sender>
   <signal>clicked()</signal>
   <receiver>DemoteStepDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup1"/>
 </buttongroups>
</ui>
